js/functions.js

```diff
@@ -95,13 +137,13 @@ function customRootCal(str) {
 }
 
 //backspace btn logic to remove last char
-function removeCharFromCal(string) {
+export function removeCharFromCal(string) {
   string = string?.substring(0, string.length - 1);
   setCharAtInputField(string);
 }
 
 //append the string at start
-function stringPreAdder(string, addString) {
+export function stringPreAdder(string, addString) {
   string == undefined ? (string = "") : (string = string);
```

Comment:
Could you please confirm if `string = string` necessary here ?

Reply:
simplified the logic with if condition.